src/main/java/io/openliberty/tools/intellij/lsp4mp4ij/psi/core/PropertiesManagerForJava.java

```diff
@@ -192,7 +192,7 @@ private void collectDefinition(String uri, PsiFile typeRoot, PsiElement hyperlin
                     // Collect all adapted definition participant
                     JavaDefinitionContext context = new JavaDefinitionContext(uri, typeRoot, utils, module,
                             hyperlinkedElement, hyperlinkedPosition);
-                    List<IJavaDefinitionParticipant> definitions = IJavaDefinitionParticipant.EP_NAME.extensions()
+                    List<IJavaDefinitionParticipant> definitions = IJavaDefinitionParticipant.EP_NAME.getExtensionList().stream()
```

Comment:
Definition of getExtensionList()
val extensionList: List<T>
    get() = getPointImpl(null).extensionList